botocore/args.py

```diff
@@ -61,6 +61,15 @@
 # values result in a warning-level log message.
 USERAGENT_APPID_MAXLEN = 50
 
+VALID_REQUEST_CHECKSUM_CALCULATION_CONFIG = (
+    "when_supported",
+    "when_required",
+)
+VALID_RESPONSE_CHECKSUM_VALIDATION_CONFIG = (
+    "when_supported",
+    "when_required",
+)
+
```

Comment:
We might consider if these should live here/if they're public. Once we expose them, it'll be harder to move/reuse these later. Importing from `args` into other modules is likely to create import cycles, so we just want to make sure we're accounting for that.

Reply:
I'll merge the PR as is for now and make a note to reevaluate the constants so we don't forget about this.